packages/starlight/components/Icons.ts

```diff
@@ -103,4 +103,6 @@ export const Icons = {
 		'<path d="M5.042 15.165a2.528 2.528 0 0 1-2.52 2.523A2.528 2.528 0 0 1 0 15.165a2.527 2.527 0 0 1 2.522-2.52h2.52v2.52Zm1.271 0a2.527 2.527 0 0 1 2.521-2.52 2.527 2.527 0 0 1 2.521 2.52v6.313A2.528 2.528 0 0 1 8.834 24a2.528 2.528 0 0 1-2.521-2.522v-6.313ZM8.834 5.042a2.528 2.528 0 0 1-2.521-2.52A2.528 2.528 0 0 1 8.834 0a2.528 2.528 0 0 1 2.521 2.522v2.52H8.834Zm0 1.271a2.528 2.528 0 0 1 2.521 2.521 2.528 2.528 0 0 1-2.521 2.521H2.522A2.528 2.528 0 0 1 0 8.834a2.528 2.528 0 0 1 2.522-2.521h6.312Zm10.122 2.521a2.528 2.528 0 0 1 2.522-2.521A2.528 2.528 0 0 1 24 8.834a2.528 2.528 0 0 1-2.522 2.521h-2.522V8.834Zm-1.268 0a2.528 2.528 0 0 1-2.523 2.521 2.527 2.527 0 0 1-2.52-2.521V2.522A2.527 2.527 0 0 1 15.165 0a2.528 2.528 0 0 1 2.523 2.522v6.312Zm-2.523 10.122a2.528 2.528 0 0 1 2.523 2.522A2.528 2.528 0 0 1 15.165 24a2.527 2.527 0 0 1-2.52-2.522v-2.522h2.52Zm0-1.268a2.527 2.527 0 0 1-2.52-2.523 2.526 2.526 0 0 1 2.52-2.52h6.313A2.527 2.527 0 0 1 24 15.165a2.528 2.528 0 0 1-2.522 2.523h-6.313Z"/>',
 	matrix:
 		'<path d="M22.5 1.5v21h-2.25V24H24V0h-3.75v1.5h2.25ZM7.46 7.95V9.1h.04a3.02 3.02 0 0 1 2.61-1.39c.54 0 1.03.1 1.48.32.44.2.78.58 1.01 1.1.26-.37.6-.7 1.03-.99.44-.28.95-.43 1.55-.43.45 0 .87.06 1.26.17.38.11.71.29.99.53.27.24.49.56.64.95.15.4.23.86.23 1.42v5.72h-2.34v-4.85c0-.29-.01-.56-.04-.8a1.73 1.73 0 0 0-.18-.67 1.1 1.1 0 0 0-.44-.45 1.6 1.6 0 0 0-.78-.16c-.33 0-.6.06-.8.19-.2.12-.37.29-.48.5a2 2 0 0 0-.23.69c-.04.26-.06.52-.06.78v4.77H10.6v-4.8l-.01-.75a2.29 2.29 0 0 0-.14-.69c-.08-.2-.23-.38-.42-.5a1.5 1.5 0 0 0-.85-.2c-.15.01-.3.04-.44.08-.19.06-.37.15-.52.28-.18.14-.32.34-.44.6-.12.26-.18.6-.18 1.02v4.96H5.25V7.94h2.21ZM1.5 1.5v21h2.25V24H0V0h3.75v1.5H1.5Z"/>',
+	openCollective:
+		'<path fill-rule="evenodd" clip-rule="evenodd" d="M21.8649 5.16556C23.2111 7.10496 24 9.46035 24 12C24 14.5397 23.2111 16.895 21.8649 18.8344L18.7564 15.726C19.3668 14.6214 19.7143 13.3513 19.7143 12C19.7143 10.6487 19.3668 9.37857 18.7564 8.27403L21.8649 5.16556ZM18.8344 2.1351L15.726 5.24357C14.6214 4.63315 13.3513 4.28571 12 4.28571C7.73952 4.28571 4.28571 7.73952 4.28571 12C4.28571 16.2605 7.73952 19.7143 12 19.7143C13.3513 19.7143 14.6214 19.3668 15.726 18.7564L18.8344 21.8649C16.895 23.2111 14.5397 24 12 24C5.37258 24 0 18.6274 0 12C0 5.37258 5.37258 0 12 0C14.5397 0 16.895 0.788937 18.8344 2.1351Z"/><path fill-rule="evenodd" clip-rule="evenodd" d="M21.8649 5.16556C23.2111 7.10496 24 9.46035 24 12C24 14.5397 23.2111 16.895 21.8649 18.8344L18.7564 15.726C19.3668 14.6214 19.7143 13.3513 19.7143 12C19.7143 10.6487 19.3668 9.37857 18.7564 8.27403L21.8649 5.16556Z"/>'
```

Comment:
Ran the SVG through [SVGOMG](https://jakearchibald.github.io/svgomg/) to compress it a little and reduce the size:

```suggestion
		'<path d="M21.86 5.17a11.94 11.94 0 0 1 0 13.66l-3.1-3.1a7.68 7.68 0 0 0 0-7.46l3.1-3.1Zm-3.03-3.03-3.1 3.1a7.71 7.71 0 1 0 0 13.51l3.1 3.11a12 12 0 1 1 0-19.73Z"/><path d="M21.86 5.17a11.94 11.94 0 0 1 0 13.66l-3.1-3.1a7.68 7.68 0 0 0 0-7.46l3.1-3.1Z"/>'
```